src/readbin/ida.ml

```diff
@@ -14,7 +16,10 @@ let run cmd =
   let r = In_channel.input_lines inp in
   In_channel.close inp; r
 
-let system cmd = Unix.system cmd |> ignore
+let system cmd =
+  let return = Sys.command cmd in
```

Comment:
Thanks for the comment, I will change the variable name and the `then()` issue. Regarding the the `get_symbols` function, I actually did check the other code using this function, and I found out that there are two places (readbin.ml, find_starts.ml) using it and both of them have the error handling process. I agree that the current `get_symbols` does not reveal the fact that it could raise exception, though. But I am not sure how to improve the code. Maybe we can discuss about it offline?